esque/topic.py

```diff
@@ -135,11 +152,52 @@ def create_topic(
             replication_factor=replication_factor,
             config=topic_config,
         )
+        return new_topic
 
-        future: Future = self.cluster.confluent_client.create_topics([new_topic])[
-            topic_name
-        ]
-        ensure_kafka_futures_done([future])
+    @raise_for_kafka_exception
+    @invalidate_cache_after
+    def create_topics(self, topics: List[NewTopic]):
+        if not topics:
+            return
+        for new_topic in topics:
+            future_list = self.cluster.confluent_client.create_topics([new_topic])
+            ensure_kafka_futures_done(list(future_list.values()))
+
+    @raise_for_kafka_exception
+    @invalidate_cache_after
+    def alter_configs(self, configs: List[ConfigResource]):
```

Comment:
Where is that ConfigResource coming from? I assume it's an internal confluent object?
Would probably nicer to merge that with Topic and TopicConfig as well.